library/CMakeLists.txt

```diff
@@ -83,13 +83,17 @@ set(src_x509
 set(src_tls
     debug.c
     net_sockets.c
+    ssl_12_cli.c
+    ssl_12_gen.c
+    ssl_12_srv.c
     ssl_cache.c
     ssl_ciphersuites.c
     ssl_cli.c
     ssl_cookie.c
     ssl_srv.c
     ssl_ticket.c
     ssl_tls.c
+    ssl_msg.c
```

Comment:
Please keep files in lexicographic order

Reply:
Fixed.